source/plugins/go/src/oms.go

```diff
@@ -1848,12 +1880,16 @@ func InitializePlugin(pluginConfPath string, agentVersion string) {
 	ContainerLogSchemaVersion := strings.TrimSpace(strings.ToLower(os.Getenv("AZMON_CONTAINER_LOG_SCHEMA_VERSION")))
 	Log("AZMON_CONTAINER_LOG_SCHEMA_VERSION:%s", ContainerLogSchemaVersion)
 
-	ContainerLogSchemaV2 = false //default is v1 schema
+	//dealing with ContainerLogSchemaVersion from configMap, default is v1 schema
+	ContainerLogSchemaV2 = false 
 
 	if strings.Compare(ContainerLogSchemaVersion, ContainerLogV2SchemaVersion) == 0 && ContainerLogsRouteADX != true {
 		ContainerLogSchemaV2 = true
 		Log("Container logs schema=%s", ContainerLogV2SchemaVersion)
 		fmt.Fprintf(os.Stdout, "Container logs schema=%s... \n", ContainerLogV2SchemaVersion)
+	} else if !IsWindows {
```

Comment:
forgive my ignorance, but how are we sure at this point of code that we are in MSI mode?

Reply:
this is mainly invoke the go routine call to fetch from DCR every 5 minutes